ChameleonForms.Example/Controllers/ExampleFormsController.cs

```diff
@@ -173,6 +173,12 @@ public IFieldConfiguration ModifyConfig(IFieldConfiguration config)
         public SomeEnum? RequiredNullableEnum { get; set; }
         public SomeEnum? OptionalEnum { get; set; }
 
+        [Required, RequiredFlagsEnum]

```

Comment:
We could update the client validation adapter to look at RequiredFlagsEnum so that we don't need to redundantly include [Required]